site/_posts/2016-10-03-jekyll-3-3-0-released.markdown

```diff
@@ -0,0 +1,90 @@
+---
+layout: news_item
+title: 'Jekyll 3.3 is here with better theme support, new URL filters, and tons more'
+date: 2016-10-03 17:08:38 -0700
+author: parkr
+version: 3.3.0
+categories: [release]
+---
+
+We have tons of new features for you in our latest release of Jekyll. Three
+key things you might want to give a whirl:
+
+### 1. Themes can now ship static & dynamic assets in an `/assets` directory
+
+We're really stoked about this one. In Jekyll 3.2, we shipped the ability
+to use a theme that was packaged as a gem. Due to security necessities and
+ease-of-use concerns, this initial ship only included support for includes,
+layouts, and sass partials. A theme couldn't write any CSS, JavaScript, or
+content to your site.
```

Comment:
1. Was it really security? I thought it was just us being iterative and getting *something* out the door to nail the basics before adding additional features.

2. I don't know what we need to spend the entire first paragraph of the post talking about what users used to *not* be able to do. Let's get to what they *can* do faster. e.g., "Back in 3.2 we shipped themes... we're adding to that today with..."

3. I like the human tone, but does the reader really care if we're stoked?